python/ray/_private/resource_spec.py

```diff
@@ -275,7 +275,11 @@ def _autodetect_num_gpus():
     """
     result = 0
     if importlib.util.find_spec("GPUtil"):
-        gpu_list = GPUtil.getGPUs()
+        try:
+            gpu_list = GPUtil.getGPUs()
+        except ValueError:
+            # gputil issue gh-26
```

Comment:
maybe we should print warning?

Reply:
I guess it depends on the goal. If all that interests us here is "is there a usable GPU", then a warning is just noise. If the goal is "the user asked for a GPU but one cannot be provided" I think it should error. My impression was this is the first case.

Reply:
Do we know more about when this call will fail? Is it any time nvidia-smi exists and the driver isn't running?

Reply:
There are two failure modes on windows (perhaps other OSes are similar) when nvidia-smi exists:
- the driver is not loaded (card detection fails at Os boot, card was removed or disabled)
- the user does not have permission to access the card